scripts/fuzz.bash

```diff
@@ -55,6 +58,15 @@ while [[ $# -gt 0 ]]; do
             shift
             shift
             ;;
+        --duration)
```

Comment:
timeout should be optional.

Reply:
It is optional. Not specifying that flag works fine, it will complain if you do specify it not followed by number.